rflx/model.py

```diff
@@ -265,6 +269,49 @@ def types(self) -> Mapping[Field, Type]:
         """Return fields and corresponding types topologically sorted."""
         return self.__types
 
+    @property
+    def graph(self) -> Dot:
+        """Return pydot graph representation of message."""
+        result = Dot(graph_name=self.full_name)
+        result.set_graph_defaults(splines='ortho', ranksep='0.8 equally')
+        result.set_edge_defaults(fontname='Fira Code', fontcolor='#6f6f6f', color='#6f6f6f')
+        result.set_node_defaults(
+            fontname='Arimo',
+            fontcolor='#ffffff',
+            color='#6f6f6f',
+            fillcolor='#009641',
+            width='1.5',
+            style='"rounded,filled"',
+            shape='box')
+        result.add_node(Node(name='Initial',
+                             fillcolor='#ffffff',
+                             shape='circle',
+                             width='0.5',
+                             label=''))
+        for f in self.__fields:
+            result.add_node(Node(name=f.name))
+        for l in self.structure:
+            result.add_edge(Edge(src=l.source.name,
+                                 dst=l.target.name,
+                                 xlabel=self.edge_label(l)))
```

Comment:
Do we want to add special handling for null messages? They do not contain a structure and thus only two unconnected nodes are plotted.

Reply:
Added in 0ff8b8a. Empty graphs are now represented by an Initial node directly connected to a Final node.